colossalai/accelerator/npu_accelerator.py

```diff
@@ -31,7 +31,7 @@ def get_version(self) -> str:
         """
         Return the version of the accelerator which torch is built against.
         """
-        return torch.version.cann
+        return torch.version.cuda
```

Comment:
This is not right as this accelerator is for NPU, not CUDA devices.

Reply:
copy that